docs/modules/develop/pages/guide_migrate_webpacker_app.adoc

```diff
@@ -187,19 +197,15 @@ Decidim.register_assets_path File.expand_path("app/packs", Rails.application.roo
 
 The deployment needs to be updated to manually run `npm install` before assets are precompiled.
 
-In the case of Capistrano this can be done with a before hook:
+In the case of Capistrano this can be done with a before hook (can be added at the end of your `config/deploy.rb` file):
 
 [source,ruby]
 ----
 namespace :deploy do
   desc "Decidim webpacker configuration"
   task :decidim_webpacker_install do
     on roles(:all) do
-      within release_path do
```

Comment:
Just curious, why don't you like `within release_path`?

Reply:
Exactly my thoughts. It seems more idiomatic doing it with the capistrano's blocks. Also would this work with any RAILS_ENV (like "staging")?

Reply:
because it didn't work on my case! while this alternative did. No idea and, sincerely, to tired to find out why.
I don't think the env has anything to do in this particular order.
However, if you are confident the "within" is better, no problem. It just didn't work in my case

Reply:
Ok I've found (sort of) the reason for this. Turns out this is a limitation on the SSHKit library, something related to escaping commands with arguments that leave the "within" keyword without effect. For more info check the official Capistrano docs: https://capistranorb.com/documentation/getting-started/tasks/

Reply:
Awesome @microstudi, indeed, it seems like that's the issue. @ferblape just a doubt, have you tried this instructions while writing the guide?

Reply:
Devil is in the details! We use that recipe a lot, but with a slightly difference I've just realized:

```
  desc "Decidim webpacker configuration"
  task :install_dependencies do
    on roles(:all) do
      within release_path do
        execute :npm, "install"
      end
    end
  end
```

It only works if you use `execute` like this. I've compared it with `execute "npm install"` and this last version doesn't work 🤯

Reply:
I remember trying something like this unsuccessfully. Let me try again to confirm

Reply:
I confirm it works, changed for ti. Thanks @ferblape

Reply:
Great! Thank you both for fixing this. I wasn't aware of this capystrano/sshkit weirdness